### DISCUSSION TITLE
Unit test

### DISCUSSION THREAD

**Discussion by marcusgardh** [DevSkillsHQ/discusssions#30] [Help] [answered]
Hi! I was wondering more exactly about 

> Unit test one module of choice. There is no need to test the whole app, as we only want to understand what you take into consideration when writing unit tests.

What kind of tests are expected here?

**Comment by fenske** [comment, marked as answer]
Hi @marcusgardh. I'm Anton from Alva. Happy to clarify.

What is meant here is an opportunity for you to demonstrate what you value when approaching unit testing of your code base. 

Produce a single unit test that would allow the hiring team to evaluate your approach to how you pick what can be unit-tested and how the test would be structured. 

Does this clarify the situation?

**Reply by marcusgardh** [reply]
I understand. So for example, I would create a new cypress test file where I write my own test to display?

**Reply by fenske** [reply]
Cypress is more for E2E testing rather than unit testing. For unit testing, a better fit would e.g. be Jest.

**Reply by fenske** [reply]
Saw you added Jest tests, so I'll close this discussion. I wish you the best of luck with the rest of your interview process!
